### PR TITLE
Bach: Support Athena Timestamp

### PR DESCRIPTION
**Pull request by KathiaBarahona**

### CODE REVIEW

**Review by jansentom — COMMENTED** [review]
Left one comment that has some consequences, but left you with a nicer solution too ;)

File: bach/bach/series/series_datetime.py [hunk 1]
```diff
@@ -492,6 +504,16 @@ def __sub__(self, other) -> 'Series':
                                           other_dtypes=tuple(type_mapping.keys()),
                                           dtype=type_mapping)
 
+    def _comparator_operation(
```
[the remark this answers is not in the document]
Comment (KathiaBarahona) [reply bach/bach/series/series_datetime.py:508]: will change this!

File: bach/tests/functional/bach/test_series_timestamp.py [hunk 2]
```diff
@@ -54,61 +67,68 @@ def test_timestamp_comparator(engine, asstring: bool):
...
-        result,
+        mt,
         # raw sqlAlchemy will give datetime with timezone UTC on some Databases, because of the used db types
         # so set use_to_pandas=True to do our data normalization in the DataFrame.to_pandas() function
         use_to_pandas=True,
-        expected_columns=['_index_skating_order', 'moment'],
+        expected_columns=[
+            '_index_skating_order', 'moment', 'eq', 'gte', 'gt', 'lte', 'lt'
+        ],
         expected_data=[
```
Comment (jansentom) [inline bach/tests/functional/bach/test_series_timestamp.py:89]: lte and gte don't have very strong tests, as they always return true

**Review by jansentom — APPROVED** [review]
Nice work, thanks, LGTM!

(Don't forget the TODOs here and in the SeriesDate tests for when time / date are both implemented)

**Pull request merged**